### PR TITLE
Show automatic update check config that was previously saved

### PR DESCRIPTION
**Pull request by divolgin**
#### What this PR does / why we need it:

- Show previously saved values when `Configure automatic updates` is opened.
- Revert non-Helm apps to use postgres for update check schedule.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
* Fixes an issue that caused custom automatic update schedule to not be shown after it's saved on the [Configure automatic updates](https://docs.replicated.com/enterprise/updating-apps#configure-automatic-updates).
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: pkg/handlers/update_checker_spec.go [hunk 1]
```diff
@@ -146,36 +131,68 @@ func (h *Handler) ConfigureAutomaticUpdates(w http.ResponseWriter, r *http.Reque
...
+		logger.Error(errors.Wrap(err, updateCheckerSpecResponse.Error))
+		JSON(w, http.StatusInternalServerError, updateCheckerSpecResponse)
 		return
 	}
 
-	JSON(w, 204, "")
+	JSON(w, http.StatusNoContent, "")
+}
+
+func (h *Handler) GetAutomaticUpdatesConfig(w http.ResponseWriter, r *http.Request) {
```
Comment (sgalsaleh) [inline pkg/handlers/update_checker_spec.go:166]: this used to be part of the application object returned to the ui when listing apps, is there a reason it's a separate handler now?

File: web/src/components/modals/AutomaticUpdatesModal.jsx [hunk 2]
```diff
@@ -97,10 +101,10 @@ export default class AutomaticUpdatesModal extends React.Component {
       selectedSchedule,
       selectedAutoDeploy,
     };
+  }
 
-    if (this.state.updateCheckerSpec === "") {
```
Comment (sgalsaleh) [inline web/src/components/modals/AutomaticUpdatesModal.jsx:101]: why is this not needed anymore? is it never empty?

**Review by sgalsaleh — APPROVED** [review]

File: web/src/components/modals/AutomaticUpdatesModal.jsx (hunk 2 above)
Comment (divolgin) [reply web/src/components/modals/AutomaticUpdatesModal.jsx:101]: yeah, there is a check above to find "custom" or "default" now

Comment (sgalsaleh) [reply web/src/components/modals/AutomaticUpdatesModal.jsx:101]: but that's for the selected schedule, not the actual cron spec being used, right?

Comment (sgalsaleh) [reply web/src/components/modals/AutomaticUpdatesModal.jsx:101]: the actual cron spec should also displayed as `@default` i believe when it's empty. the "custom" or "default" are for the drop down options

**Pull request merged**